site/static/examples/kind-with-registry.sh

```diff
@@ -29,4 +29,4 @@ ip_fmt='{{.NetworkSettings.IPAddress}}'
 cmd="echo $(docker inspect -f "${ip_fmt}" "${reg_name}") registry >> /etc/hosts"
 for node in $(kind get nodes --name "${KIND_CLUSTER_NAME}"); do
   docker exec "${node}" sh -c "${cmd}"
-done
\ No newline at end of file
+done
```

Comment:
this adds unnecessary empty space when rendered in the docs and isn't necessary to parse
for now I'd prefer to not add this

Reply:
eol should be removed now